Regression/Checksum/benchmarks_json/Langmuir_multi_psatd_single_precision.json

```diff
@@ -9,7 +9,7 @@
     "particle_weight": 127999991808.0
   },
   "lev=0": {
-    "Bx": 16.283409055462556,
+    "Bx": 16.302038904962248,
```

Comment:
I would be careful about this change and make sure you understand why this update is needed. If this is due to a change in a different PR, then the change should have been part of that PR.